README.md

```diff
@@ -77,6 +77,11 @@ https://search.maven.org/search?q=g:io.grpc%20AND%20v:1.21.0
 Development snapshots are available in [Sonatypes's snapshot
 repository](https://oss.sonatype.org/content/repositories/snapshots/).
 
+If you're running on a JDK version newer than than JDK 8, you'll also need to
+explicitly add the dependency `javax.annotation:javax.annotation-api`
+
+[javax.annotation.api]: https://mvnrepository.com/artifact/javax.annotation/javax.annotation-api
```

Comment:
We shouldn't link to mvnrepository, as it is not authoritative. We can use https://search.maven.org/artifact/javax.annotation/javax.annotation-api/ instead.